### ISSUE TITLE
Do not pass "y" to models at predict time

### ISSUE THREAD

**Issue by bjschoenfeld** [byu-dml/d3m-dynamic-neural-architecture#51]

**Comment by orionw** [comment]
It looks like this is happening implicitly and none of our models use it (obviously).  We would have to strip it from the data, before passing the data in.  Is that what you're looking for?

**Comment by bjschoenfeld** [comment]
> We would have to strip it from the data, before passing the data in

Yes. I might even go so far as to say that the models could be changed to take in x and y as separate arguments to the `fit` method, then the predict methods could just take in x.
